test/rabbit_prometheus_http_SUITE.erl

```diff
@@ -162,51 +156,36 @@ metrics_test(Config) ->
     ?assertEqual(match, re:run(Body, "TYPE", [{capture, none}])),
     ?assertEqual(match, re:run(Body, ?config(queue_name, Config), [{capture, none}])),
     %% Checking that we have the first metric from each ETS table owned by rabbitmq_metrics
-    ?assertEqual(match, re:run(Body, "rabbitmq_channel_consumers{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_channel_messages_published{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_channel_process_reductions_total{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_channel_get_ack_total{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_connection_opened_total{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_connection_incoming_bytes_total{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_connection_incoming_packets_total{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_queue_messages_published_total{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_file_descriptors_open{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_file_descriptors_open_limit{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_io_read_ops_total{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_raft_term{", [{capture, none}])),
-    ?assertEqual(match, re:run(Body, "rabbitmq_queue_messages_ready{", [{capture, none}])),
-    % There are no consumers on the queue, assert on description, not empty metric
+    ?assertEqual(match, re:run(Body, "rabbitmq_channel_consumers ", [{capture, none}])),
+    ?assertEqual(match, re:run(Body, "rabbitmq_channel_messages_published ", [{capture, none}])),
```

Comment:
This sounds like a counter

Reply:
Which metric? `rabbitmq_channel_consumers` or `rabbitmq_channel_consumers` ?

Reply:
I see 4 lines - UI fail?

OK, will convert `rabbitmq_channel_messages_published` to `rabbitmq_channel_messages_published_total` and make it a counter. I believe there are a few more than should be counters as well, like `rabbitmq_channel_messages_unroutable_dropped `, etc.